specs/darwin/fan_speed_sensors.table

```diff
@@ -1,5 +1,5 @@
 table_name("fan_speed_sensors")
-description("Fan speeds.")
+description("Fan speeds. Warning: behaves differently on Intel vs. Apple Silicon.")
```

Comment:
I thought this was just a bug, not some deep platform difference?

Reply:
For all of these, I'm not exactly sure what qualifies as bug per se. 

My thinking was more short-term vs. long term. In speaking to Sharvil, I got the sense that actually fixing these could tables require some time. So in the interim, I'm thinking about this warning. It doesn't say bug or not, necessarily. It just says, hey be careful here.

Reply:
To me, bug is something where we understand it's broken and fixable. I think fan speed falls into that category. In contrast, something like `cpuid` is intel specific, unless we get clever. 

I think almost all tables will vary across platforms. So I am broadly against adding boiler plate to each table. I might be wrong, you might interface with more end users.

Reply:
But how does this end up being actionable? I think it's a hint to someone writing SQL that they'll need to test across platforms. 

But, that's pretty much always true. Thus it's less clear to me what it adds to the schema.

Reply:
I think the thing that's missing here is that while I would expect different platforms (apple vs. windows vs. linux, etc.) to behave differently as you're saying, I think having the apple icon implies to me that it works across all MacOS.  When I ran these, results came back from all laptops and I almost missed seeing that they were inconsistent.

To your point about the boilerplate, it is true that each of these tables behaves differently in its own different way. Should we document that instead of this non-specific one-liner?